src/App.tsx

```diff
@@ -17,18 +54,33 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                filter={filter}
+                onSetQuery={setQuery}
+                onSetSelectedFilter={setSelectedFilter}
+              />
             </div>
-
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoading && (
+                <Loader />
+              )}
+              <TodoList
```

Comment:
Not important in this project, but I'll write it. 

Probably better to render the list only if it is not loading. Something like; `isLoading ? ( ... ) : <TodoList />`
But again, it isn't important

Reply:
agree with @UmizDemud

Reply:
Great! it is done :D, thanks guys!